net/strongswan/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=strongswan
 PKG_VERSION:=5.9.2
-PKG_RELEASE:=7
+PKG_RELEASE:=$(AUTORELEASE)
```

Comment:
Please don't do that in this commit. It's not related to starting charon directly from the swanctl init script.

Reply:
Yup, backed it out. It was generating the wrong value anyway (12).